dotcom-rendering/src/web/components/SendAMessage.tsx

```diff
@@ -0,0 +1,396 @@
+import { css, SerializedStyles } from '@emotion/react';
+import {
+	focusHalo,
+	neutral,
+	palette,
+	space,
+	success,
+	textSans,
+	until,
+} from '@guardian/source-foundations';
+import {
+	Button,
+	SvgChevronDownSingle,
+	SvgChevronUpSingle,
+	SvgMessageRound,
+	SvgTickRound,
+} from '@guardian/source-react-components';
+import { ErrorSummary } from '@guardian/source-react-components-development-kitchen';
+import { useState } from 'react';
+import type { MessageUsFieldType } from '../../../src/types/content';
+import { decidePalette } from '../lib/decidePalette';
+import { FormField } from './Callout/FormField';
+
+const CALLOUT_URL =
+	'https://callouts.code.dev-guardianapis.com/formstack-campaign/submit';
+
+const containerStyles = (format: ArticleFormat) => css`
+	${until.desktop} {
+		background-color: ${decidePalette(format).background.messageForm};
+		color: white;
+	}
+	background-color: ${neutral[86]};
+	padding: 12px;
+`;
+const textStyles = css`
+	${textSans.xsmall()};
+	padding-bottom: 16px;
+`;
+
+const successStyles = css`
+	${textSans.xsmall()};
+	color: white;
+`;
+
+const tickBoxStyles = css`
+	fill: ${success[400]};
+	width: 48px;
+	margin-left: -8px;
+`;
+const errorTextStyles = css`
+	color: ${palette.error[400]};
+	${textSans.medium({ fontWeight: 'bold' })};
+	display: flex;
+`;
+
+const formStyles = css`
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+`;
+
+const formFieldWrapperStyles = css`
+	display: flex;
+	flex-direction: column;
+	margin-bottom: ${space[4]}px;
+	${until.desktop} {
+		label > div,
+		label > div > span {
+			color: white;
+		}
+	}
+`;
+
+const submitButtonStyles = css`
+	${until.phablet} {
+		width: 100%;
+		flex-direction: column;
+		align-items: center;
+		justify-content: center;
+	}
+	${until.desktop} {
+		align-self: flex-start;
+		background-color: white;
+		color: black;
+	}
+	background-color: ${neutral[7]};
+	align-self: center;
+`;
+
+const footerPaddingStyles = css`
+	display: flex;
+	flex-direction: row;
+	${until.desktop} {
+		justify-content: flex-start;
+	}
+	justify-content: space-around;
+	padding-bottom: ${space[4]}px;
+`;
+
+const prefaceStyles = css`
+	${textSans.xsmall()};
+	color: 'white';
+	padding-bottom: 16px;
+`;
+
+type FormDataType = { [key in string]: any };
+
+type Props = {
+	formFields: MessageUsFieldType[];
+	formId: string;
+	format: ArticleFormat;
+	pageId: string;
+};
+
+interface FormProps extends Props {
+	submissionURL: string;
+}
+
+export const Form = ({
```

Comment:
Do we need to export this? It seems it's used only within `SendAMessage` component